Pod/ILGClasses/ILGClasses.m

```diff
@@ -66,9 +66,20 @@ + (NSSet *)subclassesOfClass:(Class)superclass
 
 + (NSSet *)classesConformingToProtocol:(Protocol *)protocol
 {
-    return [self classesPassingTest:^BOOL(Class class) {
+    NSSet *superclassesConforming = [self classesPassingTest:^BOOL(Class class) {
         return class_conformsToProtocol(class, protocol);
     }];
+    
+    NSMutableSet *subclasses = [NSMutableSet set];
+    
+    for (Class superclass in superclassesConforming) {
+        NSSet *subclassesOfSuper = [self subclassesOfClass:superclass];
+        [subclasses addObjectsFromArray:[subclassesOfSuper allObjects]];
+    }
+    
+    NSSet *allConforming = [superclassesConforming setByAddingObjectsFromSet:subclasses];
+    
+    return allConforming;
```

Comment:
Hm, good point on the o-notation - I'll see if I can figure out a way to combine the two bits into one pass. 

I'll write up some tests around the swift bit, but I believe if a class is found to conform to a protocol you can pass into this method, it'd need to respond to `conformsToProtocol:`.